bindings/python/README.md

```diff
@@ -66,16 +66,7 @@ the [Shimmer Testnet](https://api.testnet.shimmer.network), and retrieves the no
 calling [`Client.get_info()`](https://wiki.iota.org/shimmer/iota-sdk/references/python/iota_sdk/client/_node_core_api/#get_info),
 and then print the node's information.
 
-```python
-from iota_sdk import Client
-
-# Create a Client instance
-client = Client(nodes=['https://api.testnet.shimmer.network'])
-
-# Get the node info
-node_info = client.get_info()
-print(f'{node_info}')
-```
+[examples/client/getting_started.py](examples/client/getting_started.py)
```

Comment:
not sure about the introductory text above ... this could get outdated as well? Maybe remove that text?

Reply:
Fine here, what about the Rust part with the features (in the Rust README)? Should I move it into the example?

Reply:
@thibault-martinez any opinion?

Reply:
@Thoralf-M the whole point of this is to less worry about syncing the actual code with the example code, so it probably is better to move it.

Reply:
I think we should keep an introduction text, but we can maybe remove links and make it simpler. I would keep it there.